server/plugin/virtual_agent_test.go

```diff
@@ -179,7 +183,7 @@ func Test_CreatePickerAttachment(t *testing.T) {
 		response    *model.SlackAttachment
 	}{
 		{
-			description: "CreatePickerAttachment returns proper stack attachment",
+			description: "CreatePickerAttachment returns proper slack attachment",
```

Comment:
Testcases for API calls have that structure and the rest have this one. There was no specific reason for doing this but if I change all this now I will have to change it in each file which will take some time.

Do we need to make this change @manojmalik20 ?

Reply:
I don't think there's a need @raghavaggarwal2308 @ayusht2810 just because this structure also fulfills our purpose and we shouldn't spend much time on this.